build.gradle.kts

```
@@ -55,20 +44,17 @@ kotlin {
 
         val androidMain by getting {
             dependencies {
-                implementation("io.sentry:sentry-android:2.3.0")
-                implementation("androidx.core:core-ktx:1.3.1")
+                implementation("io.sentry:sentry-android:5.0.1")
             }
         }
 
         val jvmMain by getting {
             dependencies {
-                implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
-                implementation("io.sentry:sentry-core:2.3.0")
+                implementation("io.sentry:sentry-core:5.0.1")
```

```suggestion
                implementation("io.sentry:sentry:5.0.1")
```

-core does not exist anymore

---

ups 🙈

---

Fixed in https://github.com/getsentry/sentry-kotlin-multiplatform/pull/6